packages/ai-core/src/browser/window-blink-service.ts

```diff
@@ -97,8 +97,8 @@ export class WindowBlinkService {
     private async blinkDocumentTitle(agentName?: string): Promise<void> {
         const originalTitle = document.title;
         const alertTitle = agentName
-            ? `🔔 Theia - Agent "${agentName}" Completed`
-            : '🔔 Theia - Agent Completed';
+            ? nls.localize('theia/ai/core/blinkTitle/namedAgentCompleted', '🔔 Theia - Agent "{0}" Completed', agentName)
```

Comment:
I’d prefer if the emoji was placed outside the localized strings. For example:  

```ts
`🔔 nls.localize('theia/ai/core/blinkTitle/namedAgentCompleted', 'Theia - Agent "{0}" Completed', agentName)`
```

That way we avoid having emojis inside translations, since they are not guaranteed to be handled consistently across all languages (I did a quick regex search in the workspace using `\p{Extended_Pictographic}`).

I know a few strings already include emojis, mostly in the ai-* packages, but it would be good to align on this pattern. Could you please also update the `AI_CORE_PREFERENCES_TITLE` accordingly? Thanks! 🙏